qa/L0_model_config/autofill_noplatform/tensorflow_savedmodel/unknown_output/expected

```diff
@@ -1 +1 @@
-Invalid argument: unexpected inference output 'OUTPUT_UNKNOWN', allowed outputs are: OUTPUT0, OUTPUT1
\ No newline at end of file
+Invalid argument: Config file provided output name 'OUTPUT_UNKNOWN' but none found in loaded model 'unknown_output'.
```

Comment:
This message should be deferred to the validation at the model load right?
I think we fixed this in IO Refactor PR?

Reply:
You're right, this leaked in.